ui/src/global.scss

```diff
@@ -396,3 +396,17 @@ ion-modal.full-width {
     --storage-segment-3: block;
     --storage-segment-4: block;
 }
+
+ion-avatar.iconify {
+    margin-inline-end: 32px;
+    margin-top: 12px;
+    margin-bottom: 12px;
+    --border-radius: 0;
+    max-height: 24px;
+    max-width: 24px;
+}
+
+ion-avatar.iconify.icon-large {
+    max-height: 32px;
+    max-width: 32px;
```

Comment:
im always worried about fixed ratios? are we sure this works out for all display res?

Reply:
i copied these values from ion-icon